test/grappler/benchmark_cnn.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/scripts/tf_cnn_benchmarks/benchmark_cnn.py b/scripts/tf_cnn_benchmarks/benchmark_cnn.py
```

Comment:
Added a new patch file because the other patch file was created using ngraph-models and the line numbers do not match.